molgenis/molgenis-emx2
# test: refactor so tests start quicker and there is a 'testFast' option to skip slow tests

Changes:
- move all tests out of 'sql' and into 'sql-it' except the one need to initialize the database for database backed tests
- mark long running tests using @Tag("slow")
- add a 'gradle testFast' to run test excluding "slow"
- add some doc hints to explain the changes

To test:
- in intellij start a test that depends on sql database, it should start much quicker
- you can run 'gradle testFast' to only run fast test which takes about 25% of normal test time
